apps/src/dance/ai/DanceAiModal.tsx

```diff
@@ -165,13 +217,15 @@ const DanceAiModal: React.FunctionComponent = () => {
           ),
           goodEffect: JSON.parse(currentValue),
         };
+
+        minMaxAssociations.current = calculateMinMax(currentInputs);
```

Comment:
When there is an existing field value and the user 'starts over' by clicking on the AI header emoji field, do you mind walking through what happens at that point? Are we still in `INITIAL` mode?

Reply:
In that case, we jump straight to `SELECT_INPUTS` mode, as seen in the handler [here](https://github.com/code-dot-org/code-dot-org/blob/3c59f629cec4fec427bfabad290ee6bdbd54d15a/apps/src/dance/ai/DanceAiModal.tsx#L290-L298).

Reply:
Thanks!